### PR TITLE
update migration

### PR DESCRIPTION
**Pull request by trangkarol**

### CODE REVIEW

File: database/migrations/2017_03_22_010127_update_team_users.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('team_users');
```
Comment (Framgia-HoVanTuan) [inline database/migrations/2017_03_22_010127_update_team_users.php]: Drop?

File: database/migrations/2017_03_22_011849_update_position_teams.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('position_teams');
```
Comment (Framgia-HoVanTuan) [inline database/migrations/2017_03_22_011849_update_position_teams.php]: Drop?

File: database/migrations/2017_03_22_011930_update_skill_users.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('skill_users');
```
Comment (Framgia-HoVanTuan) [inline database/migrations/2017_03_22_011930_update_skill_users.php]: Drop?

File: database/migrations/2017_03_22_011959_update_project_teams.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('project_teams');
```
Comment (Framgia-HoVanTuan) [inline database/migrations/2017_03_22_011959_update_project_teams.php]: Drop?

File: database/migrations/2017_03_22_011849_update_position_teams.php [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+         Schema::rename('position_teams', 'position_team');
```
Comment (at-diephq) [inline database/migrations/2017_03_22_011849_update_position_teams.php]: sai indent

**Pull request merged**